packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-chip.js

```diff
@@ -42,6 +42,25 @@ class MultiSelectComboBoxChip extends ThemableMixin(PolymerElement) {
 
   static get template() {
     return html`
+      <style>
+        :host {
+          display: inline-flex;
+          align-items: center;
+          align-self: center;
+          white-space: nowrap;
+          box-sizing: border-box;
+          min-width: 0;
```

Comment:
This is needed to ensure long chips shrink properly, so I moved it from Lumo to `src`.